pkg/controller/plan/kubevirt.go

```diff
@@ -1708,7 +1708,7 @@ func (r *KubeVirt) guestConversionPod(vm *plan.VMStatus, vmVolumes []cnv.Volume,
 	nonRoot := true
 	allowPrivilageEscalation := false
 	// virt-v2v image
-	coldLocal, vErr := r.Context.Plan.VSphereColdLocal()
+	coldLocal, vErr := r.Context.Plan.ShouldUseV2vForTransfer()
```

Comment:
nit: maybe worth renaming the "coldLocal" variable to match the new method name.